dist/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/a0d360eeab.js" crossorigin="anonymous"></script>
+  <title>To-Do List</title>
+</head>
+
+<body>
+  <header class="heading-container">
+    <h1>To Do</h1>
+  </header>
+  <main class="project-container">
+    <aside class='side-bar'>
+      <div class='d-flex justify-content-around'>
+        <h2 class="project-heading">Add Projects<i class="fa fa-plus" aria-hidden="true" id="add-project"></i></h2>
+        <!-- <h2 id='view-project'>View All <i class="fa fa-arrow-right" aria-hidden="true"></i></h2> -->
```

Comment:
## Remove commented out code
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history just as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/), thank you!